examples/oneapi/cpp/source/spectral_clustering/spectral_clustering_pipeline.cpp

```diff
@@ -0,0 +1,90 @@
+/*******************************************************************************
+* Copyright contributors to the oneDAL project
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "example_util/utils.hpp"
+#include "oneapi/dal/algo/kmeans.hpp"
+#include "oneapi/dal/algo/kmeans_init.hpp"
+#include "oneapi/dal/algo/spectral_embedding.hpp"
+#include "oneapi/dal/io/csv.hpp"
+#include <algorithm>
+#include <math.h>
```

Comment:
```suggestion
#include <cmath>
```

Please use a [regular C++ headers](https://en.cppreference.com/w/cpp/header/cmath)